ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -490,26 +491,19 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
         rp.populationHealth || rp.socialBehavioral;
     }
 
-    get noSpecificPopulationSelected() {
-      return this.state.workspace.researchPurpose.population &&
-        (!this.state.workspace.researchPurpose.populationDetails ||
-          this.state.workspace.researchPurpose.populationDetails.length === 0);
-    }
-
-    get noDiseaseOfFocusSpecified() {
-      return this.state.workspace.researchPurpose.diseaseFocusedResearch &&
-        !this.state.workspace.researchPurpose.diseaseOfFocus;
+    get specificPopulationSelected() {
```

Comment:
👍 for this PR generally - I made a half-hearted attempt at this once but got distracted.  Thanks for fixing.

But I prefer the previous semantics for these functions.  If `population` is not selected, this will validate correctly but the function name won't be accurate.

Reply:
That makes sense, I can revert it and invert the truthiness expectation.

Reply:
nit: how about adding `is` as a prefix to indicate the boolean return value?

Reply:
@jmthibault79 would you consider this to also solve your remark?

Reply:
Only adding "is" would not address my concern.  Renaming the original with "is" would be fine - maybe `isSpecificPopulationSelectionMissing()` ?

Reply:
oh, regarding the truthiness, I generally prefer `isTrue` version over the inverse unless there's a very intuitive reason for the inverse.

Reply:
That is why I changed it.

Reply:
That is my default preference as well - but I think in this case it would be slightly misleading.

Reply:
I am finding it confusing in this case.